jobfunnel/backend/scrapers/indeed.py

```diff
@@ -354,3 +356,42 @@ def _get_search_url(self, method: Optional[str] = 'get') -> str:
             raise NotImplementedError()
         else:
             raise ValueError(f'No html method {method} exists')
+
+
+class IndeedScraperFRFre(BaseIndeedScraper, BaseFRFreScraper):
+    """Scrapes jobs from www.indeed.fr
+    """
+    def _get_num_search_result_pages(self, search_url: str, max_pages=0) -> int:
```

Comment:
perhaps we can put this into the BaseIndeedScraper? Just wondering if there are meaningful differences between them or if we can just share them all.

Reply:
There are meaningful differences in this case. The reason is that indeed.fr is in French so that text can be presented differently. However, div tags and ids are identical to the English websites.

Reply:
Howe about we make a shared method `get_num_search_result_pages` in the `BaseIndeedScraper`, and we use `self.locale` to determine how the text should be handled?

Reply:
Good point. Since we know that Indeed does not alter the html divs, classes, etc we can share all logic that handles these html elements. For the localised scrapers we could simply things by only focusing on the actual texts within these HTML elements.